src/main/ApplicationUtils.cpp

```diff
@@ -662,7 +672,6 @@ publish(Application::pointer app)
 std::string
 minimalDBForInMemoryMode(Config const& cfg)
 {
-    return fmt::format(FMT_STRING("sqlite3://{}/minimal.db"),
-                       cfg.BUCKET_DIR_PATH);
+    return fmt::format(FMT_STRING("sqlite3://{}"), minimalDbPath(cfg));
```

Comment:
note that if you return a path, you'll have to use `generic_string()` here as you want `/` on all platforms